packages/page-calendar/src/DayItem.tsx

```diff
@@ -46,62 +50,116 @@ function DayItem ({ className, item: { date, info, type } }: Props): React.React
                     : ['treasurySpend'].includes(type)
                       ? <div className='itemLink'><a href='#/treasury'>{t<string>('via Treasury')}</a></div>
                       : undefined;
+      let s = '';
 
       switch (type) {
         case 'councilElection':
-          return <><div className='itemDesc'>{t<string>('Election of new council candidates')}</div>{typeLink}</>;
-
+          s = 'Election of new council candidates';
```

Comment:
So on these we need to bring back the `t<string>('Election of new council candidates')` wrapper - that allows translation. Without it (or being dynamic), the i18n functionality is lost. The output can still be uses (i.e. the result from the `t(...)`) in the iCal, it is just a string.

TL;DR For all of these, just remove the fragment/div wrappers, keep the `t(...)`